swagger/src/main/scala/org/scalatra/swagger/Swagger.scala

```diff
@@ -109,15 +120,24 @@ object Swagger {
       val fields = klass.erasure.getDeclaredFields.toList collect {
         case f: Field if f.getAnnotation(classOf[ApiModelProperty]) != null =>
           val annot = f.getAnnotation(classOf[ApiModelProperty])
-          val asModelProperty = toModelProperty(descr, Some(annot.position()), annot.required(), annot.description().blankOption, annot.allowableValues())_
+          val asModelProperty = toModelProperty(
+            descr = descr,
+            position = Some(annot.position()),
+            required = annot.required(),
+            description = Option(annot.description()),
+            allowableValues = annot.allowableValues(),
+            example = Option(annot.example()),
+            default = Option(annot.defaultValue())
+          )_
```

Comment:
last mention for formatting changes too. even if this does match their style guide I would avoid to change unless it comes from upstream.